mappings/net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig.mapping

```diff
@@ -1,5 +1,8 @@
 CLASS net/minecraft/class_2906 net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig
+	FIELD field_13221 seaLevel I
 	FIELD field_13222 forcedBiome I
+	FIELD field_13223 riverSize I
+	FIELD field_13224 biomeSize I
```

Comment:
```suggestion
	FIELD field_13222 forcedBiome I
```
See #1123. Insufficient evidence for these.

Reply:
understood